src/assembler/parser/index.ts

```diff
@@ -78,6 +80,24 @@ export function parse(code: string): ICodeFile {
       name: 'Comment',
       pattern: COMMENT
     },
+    {
+      name: 'spaceOrFillInstruction',
+      pattern: SPACE_OR_FILL_INSTRUCTION,
+      onMatch(match: ITextMatch) {
+        if (!area) {
+          throw new ParseError(
+            'SPACE or FILL must be defined in area',
+            match.from
+          )
+        }
+        const instruction: IInstruction = {
+          name: match.captures[0].toUpperCase(),
```

Comment:
Map % to SPACE or is this handled on encoding?

Reply:
encoder does it already 
![grafik](https://user-images.githubusercontent.com/56474048/169249564-47abcfd1-7c51-4884-b604-ef4625ed7363.png)